components/camel-ftp/src/main/java/org/apache/camel/component/file/remote/SftpConfiguration.java

```diff
@@ -65,6 +65,8 @@
     private String preferredAuthentications;
     @UriParam(defaultValue = "WARN")
     private LoggingLevel jschLoggingLevel = LoggingLevel.WARN;
+    @UriParam(label="advanced", description="Specifies how many requests may be outstanding at any one time.")
```

Comment:
Only document one place, eg the javadoc. So remove the description. Also put space around label="ad..."